packages/pipeline-editor/src/RuntimesWidget.tsx

```diff
@@ -92,6 +92,11 @@ class RuntimesDisplay extends MetadataDisplay<
           <br />
         </span>
       );
+    } else if (metadata.schema_name === 'kfp') {
```

Comment:
there's no reason to make this an else if, it can just be a separate if statement. Also it could potential have issues if `metadata_props` is defined despite it being a kfp schema

Reply:
Shouldn't we be migrating to use `metadata.runtime_type` rather than the specific schema name?  I see this change is consistent with the detection of `airflow` above, but my comment applies there as well.

I suppose if someone brought their own KUBEFLOW_PIPELINES runtime, their schema may not have this particular property, so, in that sense, this may be more correct (technically).  As a result, I guess I retract my comment but am still posting here to bring attention to when `schema_name` vs. `runtime_type` should be used.

Reply:
Fixed